travis/check_tags.py

```diff
@@ -0,0 +1,130 @@
+    # list of tags from match_tags_commit
+    list_tags = re.findall(r'(:\w+:)', match_tags_commit)
+    # list of tags that should not be in the commit
+    extra_tags = [i for i in list_tags if i not in DEVELOPMENT_TAGS + RELEASE_TAGS + VERSION_TAGS]
+    if extra_tags != []:
+        error = {commit: 'There should not be such tags in the commit!'}
+        errors_commit.update(error)
+        return errors_commit
+    # lists of Development tag and Release tag in commit
+    dev_tag = list(set(list_tags) & set(DEVELOPMENT_TAGS))
+    release_tag = list(set(list_tags) & set(RELEASE_TAGS))
+    if symbol_in_branch:
+        errors_dev = check_dev_branch_tags(dev_tag, commit, list_tags)
+        errors_commit.update(errors_dev)
+    else:
+        errors_stable = check_stable_branch_tags(release_tag, commit, list_tags)
+        errors_commit.update(errors_stable)
+
+    if any(tag in REQUIREMENTS_TAGS_OF_VERSION for tag in list_tags):
+        errors_version = check_version_tags(list_tags, commit, version)
+        errors_commit.update(errors_version)
+    return errors_commit
+
+
+def check_version_tags(list_tags, commit, version):
+    errors_version = {}
+    if not any(tag in VERSION_TAGS for tag in list_tags):
+        error = {commit: 'Must be Version tags!'}
+        errors_version.update(error)
+        return errors_version
+    # list of digit from tag's of commit
+    list_digits = [x.replace(':', '') for x in list_tags if x in VERSION_TAGS]
+    version_in_commit = ''
+    for digit in list_digits:
+        # calculates version in commit
+        version_in_commit += VERSION_TAGS_DICT.get(digit)
+    if version.replace(".", "") != version_in_commit:
+        error = {commit: 'Version in commit is wrong!'}
+        errors_version.update(error)
+        return errors_version
+    # list of indices (requirements of version's tags in commit)
+    index_requirements_tags_version = [list_tags.index(i) for i in list_tags if i in REQUIREMENTS_TAGS_OF_VERSION]
+    # list of indices (version's tags in commit)
+    index_verion_tags = [list_tags.index(i) for i in list_tags if i in VERSION_TAGS]
+    # Check proper order of tags in commit: comparison indices "requirements of versions tags" and "versions tags"
+    if not index_requirements_tags_version[-1] < index_verion_tags[0]:
+        error = {commit: 'Version tag must be after the main tag!'}
+        errors_version.update(error)
+        return errors_version
+    return errors_version
+
+
+def check_dev_branch_tags(dev_tag, commit, list_tags):
+    errors_dev = {}
+    if dev_tag == []:
+        error = {commit: 'There should be one Development tag in the dev branches!'}
+        errors_dev.update(error)
+        return errors_dev
+    # checking the number of dev tags in commit
+    if len(dev_tag) > 1:
+        error = {commit: 'You must use only one Development tag!'}
+        errors_dev.update(error)
+        return errors_dev
+    if any(tag in RELEASE_TAGS for tag in list_tags):
```

Comment:
Move this block BEFORE checking ``if dev_tag == []`` to have more specific error message when release tag was used instead of dev.

Compare:

Current: 

> There should be one Development tag in the dev branches!'

If you move BEFORE:

> For dev branches must be Development tags only!

Otherwise You message "For dev branches must be Development tags only!" will be printed only in rare case when developer used one dev and one release tag